src/components/transaction/details/TransactionBody.tsx

```diff
@@ -0,0 +1,99 @@
+                        ARoka...uRKma
+                    </span>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.AMOUNT')}>
+                    <AmountBadge amount={'+14.56128396 ARK'} type='positive' />
+                    <span className='pl-0.5 text-theme-secondary-500 dark:text-theme-secondary-300'>
+                        $0.50
+                    </span>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.TRANSACTION_FEE')}>
+                    0.75 ARK{' '}
+                    <span className='text-theme-secondary-500 dark:text-theme-secondary-300'>
+                        $0.50
+                    </span>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.TIMESTAMP')}>27 Mar 2024, 15:01:04</TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.TRANSACTION_ID')}>
+                    <div className='flex w-full flex-row items-center justify-between'>
+                        <span>231d80d0c255a...fc4a6019815773f0</span>
+                        <button
+                            type='button'
+                            className='block'
+                            onClick={() =>
+                                copy('231d80d0c255afc4a6019815773f0', t('COMMON.TRANSACTION_ID'))
+                            }
+                        >
+                            <Icon
+                                icon='copy'
+                                className='h-5 w-5 text-theme-primary-700 dark:text-theme-primary-650'
+                            />
+                        </button>
+                    </div>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.MEMO')}>
+                    <span className='text-theme-secondary-500 dark:text-theme-secondary-300'>
+                        N/A
+                    </span>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.VOTE')}>
+                    boldninja{' '}
+                    <span className='text-theme-secondary-500 dark:text-theme-secondary-300'>
+                        DKrAC...fXYqu
+                    </span>
+                </TrasactionItem>
+
+                <TrasactionItem title={t('COMMON.UNVOTE')}>
+                    genesis_31{' '}
+                    <span className='text-theme-secondary-500 dark:text-theme-secondary-300'>
+                        DJmvh...K3yLt
+                    </span>
+                </TrasactionItem>
+            </div>
+            <div>
+                <ExternalLink
+                    href={getExplorerDomain(
+                        primaryWallet?.network().isLive() ?? false,
+                        primaryWallet?.address() ?? '',
+                    )}
+                    className='hover:no-underline'
+                >
+                    <Button variant='secondary'>{t('COMMON.VIRE_ON_ARKSCAN')}</Button>
```

Comment:
also update the i18n file

```suggestion
                    <Button variant='secondary'>{t('COMMON.VIEW_ON_ARKSCAN')}</Button>
```